_includes/people.html

```diff
@@ -51,7 +51,7 @@
     </div>
 
     {% if ols-roles and ols-roles != '' or user.title %}
-    <p class="people-description">
+    <p class="people-description" style="height: 150px;">
```

Comment:
Can you remove the height here as it is in the CSS? And check everything if fine :smile: